python/pyxbos/pyxbos/drivers/obvius/obvius.py

```diff
@@ -0,0 +1,198 @@
+from pyxbos.driver import *
+import logging
+import os, re, io
+import requests
+import time
+import urllib
+import yaml
+
+from bs4 import BeautifulSoup as bs
+from datetime import datetime, timedelta
+import sensordb
+import pandas as pd
+
+try:
+    import ordereddict
+except ImportError:
+    import collections as ordereddict
+
+class ObviusDriver(Driver):
+    def setup(self, cfg):
+        self.hostname = os.uname()[1]
```

Comment:
Can we try for a more meaningful base resource for the Obvius driver? Maybe some URL-safe transformation of the building or site name?

Reply:
Oh yes, good point. I'm assuming you're referring to `self.hostname = os.uname()[1]` (line 21). Info about the operating system is likely not particularly useful haha.

Sounds good, I will change the hostname to be assigned to something that refers to the building or site name, something I can grab from the `cfg` parameter :)

Reply:
Just making note of it here after our in person sync, will delete this field since it serves no purpose. 👍

Reply:
Resolved, `hostname` field  has been removed from file entirely.